data_diff/database.py

```diff
@@ -714,6 +716,65 @@ def normalize_value_by_type(self, value: str, coltype: ColType) -> str:
 HELP_PRESTO_URI_FORMAT = "presto://<user>@<host>/<catalog>/<schema>"
```

Comment:
Consider upper-casing the SCHEMA and WAREHOUSE in the help string for Snowflake as a nudge 👍🏻

Reply:
Good idea